K2Bridge.Tests.End2End/flights/MSearch_Invalid_Query.json

```diff
@@ -9,14 +9,11 @@
       }
     }
   ],
-  "_source": {
-    "excludes": []
-  },
   "aggs": {
     "2": {
       "date_histogram": {
         "field": "timestamp",
-        "interval": "1M",
+        "calendar_interval": "month",
```

Comment:
for other reviewers.
I made sure to have both calendar_interval and fixed_interval as both of these replace the old 'interval'. making sure we  support both